.github/workflows/create-update-issues.yaml

```diff
@@ -0,0 +1,80 @@
+name: Create Update Issues
+
+on:
+  workflow_call:
+    inputs:
+      AUTH_APP_ID:
+        description: Github app id to authenticate in mobile and extention repositories
+        type: string
+        required: true
+    secrets:
+      AUTH_APP_PRIVATE_KEY:
+        description: Github app private key to authenticate in mobile and extention repositories
+        required: true
+
+jobs:
+  create-update-issues:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout head
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get release tag
+        id: releaseTag
+        run: |
+          echo "tag=$(git describe --tags --abbrev=0)" >> "$GITHUB_OUTPUT"
+        shell: bash
+
+      - name: Check released package
+        id: releaseCheck
+        env:
+          tag: ${{ steps.releaseTag.outputs.tag }}
+        run: |
+          if [[ "${tag}" == @metamask/*  ]] ; then
+            # Extract package name without the leading '@'
+            package_name="${tag#@}"
+            package_name="${package_name%@*}"
+            echo "package_name=${package_name}" >> "$GITHUB_OUTPUT"
+
+            # Extract version number
+            version="${tag##*@}"
+            echo "version=${version}" >> "$GITHUB_OUTPUT"
+
+            # Check if version number ends with .0.0
+            if [[ $version == *.0.0 ]]; then
+              echo "$package_name major update"
+              echo "major_version=true" >> "$GITHUB_OUTPUT"
+            else
+              echo "$package_name minor update"
+              echo "major_version=false" >> "$GITHUB_OUTPUT"
+            fi            
+          else
+            echo "Monorepo version update"
+            echo "major_version=false" >> "$GITHUB_OUTPUT"
+          fi
+        shell: bash
+
+      - name: Generate a token
+        id: generate-token
+        if: ${{ steps.releaseCheck.outputs.major_version == 'true' }}
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ inputs.AUTH_APP_ID }}
+          private-key: ${{ secrets.AUTH_APP_PRIVATE_KEY }}
+          repositories: metamask-mobile, metamask-extension
+
+      - name: Create Issue
+        if: ${{ steps.releaseCheck.outputs.major_version == 'true' }}
+        env:
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+          package: ${{ steps.releaseCheck.outputs.package_name }}
+          version: ${{ steps.releaseCheck.outputs.version }}
+        run: |
+          echo "Creating new issue..."
+          gh issue create --title "Update ${package} to version ${version}" --body "Please update ${package} to version ${version}" --repo "MetaMask/metamask-extension"
```

Comment:
We'll want to assign teams as well, this was part of the acceptance criteria and is important for achieving the main goal here (visibility and accountability for these updates).

We use labels to keep track of who owns which issue, so assigning a team here means attaching that team's label to the issue (which luckily is easy with the `gh issue` command).

However, determining which team to assign might be tricky. The ticket suggests that we assign it to "the responsible team", but doesn't describe who should be responsible. In discussing this work, we had considered either:
* Assigning the team who released the work on core, or
* Assigning the "codeowner" team(s) for the given package.

I think either way would be fine to start with, as the assigned team can easily re-assign a different team as appropriate. As long as some team is assigned, it accomplishes the goals of visibility and accountability.

For the first strategy, we could get the team label using an approach like this: https://github.com/MetaMask/github-tools/blob/058012b49ff2fbd9649c566ba43b29497f93b21d/.github/workflows/add-team-label.yml#L19
 (we already have a mapping of user to team in our planning repo)
 
 For the second strategy, we could parse the `.github/CODEOWNERS` file. But we'd need a mapping of GitHub team names to labels (the format is a little different between the two right now, and doesn't follow a consistent naming convention from team to team).